puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
It looks like you are missing passing these params to quickstack::cinder, and whatever changes you need in quickstack::cinder itself.  For OSP 6, the pacemaker/ manifests and whatever they call are what is officially supported.  So, as is, this may not throw an error, but it will certainly not do what you want if you are using Staypuft with this.